ide-common/src/main/kotlin/org/digma/intellij/plugin/auth/AuthManager.kt

```diff
@@ -34,6 +33,8 @@ import java.util.concurrent.locks.ReentrantLock
 const val SILENT_LOGIN_USER = "admin@digma.ai"
 const val SILENT_LOGIN_PASSWORD = "admin"
 
+data class LoginResult(val isSuccess: Boolean, val userId: String?, val error: String?)
```

Comment:
looks a bit like duplication of org.digma.intellij.plugin.model.rest.login.LoginResponse, maybe it can be used instead or this login response can wrap org.digma.intellij.plugin.model.rest.login.LoginResponse.
but if that's easier then not such a big deal